tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt

```diff
@@ -1,3 +1,16 @@
+# Copyright (c) 2022 by Hurleveur, an anonymous author
```

Comment:
Why did you add anonymous author here but not for the rest? Not that it matters much but we should prob have conventions (or does it mean me and an anonymous author?)

Reply:
You and someone else.
I think @UmorderedSigh was involved in writing this file? This is how she refers to herself in copyright headers. Assuming she wants to be listed here.

Reply:
I think it would be clearer with an and in that case
Also we dont really have to mention amonymous authors?

This doesnt matter much either way, it just feels weird

Reply:
I'm still an author, whether I'm anonymous or not.

Reply:
Yeah so are everyone adding content to all files, we just dont mention them anyway, and I never noticed by an anonymous author, declining copyright often goes to MZ or whoever else wrote the file?
This doesnt matter much my issue with it was mosty the confusion that it said `Hurleveur, an anonymous author`